view/rooms/app.js

```diff
@@ -660,7 +660,8 @@ function setupSocketIOClient(bots) {
     botInfo[bot.body.id] = bot.body.name;
   });
 
-  const socket = _io('/', { transports: ['websocket'] });
+  const namespace = `/refocus.room?room=${ROOM_ID}&roomtype=${_roomTypeName}`;
```

Comment:
@tausifmuzaffar not important but should we use '&roomType' here as well to stick with the naming convention?